src/main/kotlin/net/ccbluex/liquidbounce/features/misc/proxy/ProxyValidator.kt

```diff
@@ -54,8 +54,16 @@ import kotlin.jvm.optionals.getOrNull
 /**
  * This is a generic Minecraft server that is used to check if a proxy is working. The server also
  * responds to query requests with the client's IP address.
+ * Some of these are taken from https://topminecraftservers.org/
  */
-private const val PING_SERVER = "ping.liquidproxy.net"
+private val FALLBACK_PING_SERVERS = listOf(
+    "mc.hypixel.net",
+    "crescentkingdom.com",
+    "gratopia.gg",
+    "terranova.mc.gg",
+    "play.arcanecloud.net",
+    "ping.liquidproxy.net"
+)
```

Comment:
No, we will not use this.

Reply:
Oh. But what about the server list idea? Aren't we supposed to reuse the server list as ping servers? because I can tell you that no one would want to join a server which shows their IP as the MOTD or the kick message.

Reply:
What are you talking about?